stable/vulcan/values.yaml

```diff
@@ -376,11 +376,9 @@ api:
     bucketLogs: logs
     s3PrivateBucket: insights
     s3PublicBucket: public-insights
-    queueArn: arn:aws:sqs:local:012345678900:VulcanK8SAPIScans
     queueName: VulcanK8SAPIScans
```

Comment:
Remove also queueName. I think it was not used.

Reply:
👍